### PR TITLE
add khavee to pythainlp

### PR DESCRIPTION
**Pull request by kangkengkhadev**
### What does this changes

Brief summary of the changes

### What was wrong

Description of what was the root cause of the issue.

### How this fixes it

Description of how the changes fix the issue.

Fixes #...

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/PyThaiNLP/pythainlp/blob/dev/CONTRIBUTING.md) to this repository.

- [ ] Passed code styles and structures
- [ ] Passed code linting checks and unit test

### CODE REVIEW

**Comment by kangkengkhadev** [comment]
Add Khavee the poem verifier to Pythainlp

File: pythainlp/khavee/__init__.py
Comment (wannaphong) [inline pythainlp/khavee/__init__.py:1]: Can you add

```python
__all__ = ["KhaveeVerifier"]

from pythainlp.khavee.core import KhaveeVerifier
```

**Review by wannaphong — CHANGES_REQUESTED** [review]
Thank you for sent this pull request! I have a comment that you can edit before I will merging this  pull request.

**Review by wannaphong — APPROVED** [review]

**Comment by wannaphong** [comment]
Thank you!

**Pull request merged**